core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -122,6 +122,16 @@ public AbstractClient(ClientContext context, InetSocketAddress address,
    */
   protected abstract ServiceType getRemoteServiceType();
 
+  /**
+   * Gets the remote service version.
+   * <p>
+   * Returns a long value representing the version for the remote service
+   * utilizing the {@link #mVersionService}.
+   *
+   * @return  the remote service version
+   * @throws  AlluxioStatusException  if an unforeseen exception occurs while
```

Comment:
The expression "unforeseen  exception" is not clear. The exception is known, what actually throws it is unknown.